src/libs/routes/notFoundRoute.ts

```diff
@@ -0,0 +1,8 @@
+import { Request, Response, NextFunction } from 'express'
+
+export default (req, res, next) => {
+    next({
+        error : "Not Found",
+        code: 404
+    })
+}
```

Comment:
Give proper name to function

Reply:
I forgot to add function but I will give proper function name  in this particular files.